rocAL/source/loaders/audio/audio_loader_sharded.cpp

```diff
@@ -48,6 +48,13 @@ AudioLoaderSharded::~AudioLoaderSharded() {
     _loaders.clear();
 }
 
+size_t AudioLoaderSharded::last_batch_padded_size() {
+    size_t sum = 0;
+    for (auto& loader : _loaders)
+        sum += loader->last_batch_padded_size();
```

Comment:
Should we get the last_batch_padded_size from all shards and return the sum?

Reply:
No thats not needed. This function is independent of the shards